website/src/routes/guides/(migration)/migrate-from-zod/index.mdx

```diff
@@ -1,7 +1,101 @@
 ---
 title: Migrate from Zod
+description: Im based on Zod, so migrating should not require much work, because the naming and behaviour of methods is very similar.
 ---
 
+import { Link } from '@builder.io/qwik-city';
+
 # Migrate from Zod
 
-> The content of this page is not yet ready. Check back in a few days.
+I am heavily based on Zod, you could say it was the inspiration for my own creation. That's why a migration should not require much work, because the naming and behaviour of methods is very similar.
+
+## Setup
+
+First you need to install me. See <Link href="/guides/installation/">Installation instructions</Link>.
+
+## Schema Migration
+
+After this setup you will have to change every **Zod Schema**,
+but you don't have to change all of them just yet.
+
+> **This is a Process, so take your time!** *You can always take a break, and I will still be here, waiting for you.*
+
+Theres usually two ways to write these Schemas with Zod:
+
+### The `z` Object
+
+> Zod recommends the use of it's `z` object export, which contains every schema class. Modern bundlers can remove ([treeshake](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking)) unused functions and objects, but this does not work with JavaScript classes. Zod Schemas and validation chains are build using class methods,  this is one of it's disadvantages compared to me. Learn more <Link href="/guides/comparison/">here</Link>.
```

Comment:
> Zod recommends the use of it's `z` object export,

Is it actually true? I do not see any reason since `import * as z from 'zod'` will produce slightly smaller bundle. Maybe we should not talk about Zod too much since this part can be also out of date quickly. I think the remaining sentences make sense to me but let me try to potentially improve:

`Tree-shaking does not work well with JavaScript classes. Zod schemas and validation chains are built using class methods, which is one of its disadvantages compared to me in terms of the final bundle size`

How about this?